sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -20,6 +20,7 @@ package org.apache.spark.sql.hive
 import java.util.Properties
 
 import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
```

Comment:
Shall we remove this, @StefanXiepj ?

Reply:
Thanks for your review, i have removed it.